.github/workflows/ci.yml

```diff
@@ -23,47 +16,66 @@ jobs:
 
-      - name: Setup Java (temurin@11)
-        if: matrix.java == 'temurin@11'
-        uses: actions/setup-java@v2
-        with:
-          distribution: temurin
-          java-version: 11
-
-      - name: Cache sbt
-        uses: actions/cache@v2
+      - name: Setup Java
+        uses: actions/setup-java@v3
         with:
-          path: |
-            ~/.sbt
-            ~/.ivy2/cache
-            ~/.coursier/cache/v1
-            ~/.cache/coursier/v1
-            ~/AppData/Local/Coursier/Cache/v1
-            ~/Library/Caches/Coursier/v1
-          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
-
-      - name: Check that workflows are up to date
-        run: sbt ++${{ matrix.scala }} githubWorkflowCheck
+          distribution: ${{ matrix.java.distro }}
+          java-version: ${{ matrix.java.version }}
+          cache: sbt
 
       - name: Build project
-        run: sbt ++${{ matrix.scala }} test
+        run: sbt test
 
       - name: Compress target directories
-        run: tar cf targets.tar joda/target/1-16-jvm-2.13 joda/target/1-16-jvm-3 core/target/1-15-jvm-2.13 core/target/1-16-js-3 joda/target/1-13-jvm-2.12 core/target/1-16-native-2.13 core/target/1-16-native-3 core/target/1-13-jvm-2.12 core/target/1-15-jvm-2.12 core/target/1-15-js-2.12 joda/target/1-15-jvm-3 joda/target/1-15-jvm-2.13 core/target/1-16-jvm-3 core/target/1-15-js-2.13 joda/target/1-14-jvm-2.13 target core/target/1-15-jvm-3 core/target/1-15-js-3 joda/target/1-14-jvm-2.12 core/target/1-16-js-2.12 core/target/1-15-native-3 core/target/1-16-jvm-2.13 joda/target/1-16-jvm-2.12 joda/target/1-15-jvm-2.12 core/target/1-16-js-2.13 core/target/1-16-jvm-2.12 core/target/1-14-jvm-2.12 core/target/1-14-jvm-2.13 core/target/1-15-native-2.13 project/target
+        run: |
+          tar cf targets.tar \
+          core/target/1-13-jvm-2.12 \
+          core/target/1-14-jvm-2.12 \
+          core/target/1-14-jvm-2.13 \
+          core/target/1-15-js-2.12 \
+          core/target/1-15-js-2.13 \
+          core/target/1-15-js-3 \
+          core/target/1-15-jvm-2.12 \
+          core/target/1-15-jvm-2.13 \
+          core/target/1-15-jvm-3 \
+          core/target/1-15-native-2.13 \
+          core/target/1-15-native-3 \
+          core/target/1-16-js-2.12 \
+          core/target/1-16-js-2.13 \
+          core/target/1-16-js-3 \
+          core/target/1-16-jvm-2.12 \
+          core/target/1-16-jvm-2.13 \
+          core/target/1-16-jvm-3 \
+          core/target/1-16-native-2.13 \
+          core/target/1-16-native-3 \
+          joda/target/1-13-jvm-2.12 \
+          joda/target/1-14-jvm-2.12 \
+          joda/target/1-14-jvm-2.13 \
+          joda/target/1-15-jvm-2.12 \
+          joda/target/1-15-jvm-2.13 \
+          joda/target/1-15-jvm-3 \
+          joda/target/1-16-jvm-2.12 \
+          joda/target/1-16-jvm-2.13 \
+          joda/target/1-16-jvm-3 \
+          project/target \
+          target
```

Comment:
Would be nice if you didn't need all that...

Reply:
Technically, we don't... but I was copying the ci.yml file generated automatically by the `sbt-github-actions` and `sbt-ci-release` plugin. My hope is to better integrate these plugins with `sbt-projectmatrix` so that I don't have to maintain this manually.